UpCloud Managed Object Storage.cyberduckprofile

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2018 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>upcloud</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>UpCloud Managed Object Storage</string>
+        <key>Default Hostname</key>
```

Comment:
Use `Hostname Placeholder` instead.

Reply:
This works but I find the profile hostname now defaults to `s3.amazonaws.com` upon adding it, which I don't want. I tried adding the `Default Hostname` key back with an empty string but that didn't change the behaviour. Do you have any other suggestion? Basically we are using unique hostnames for each user instance so we cannot pre-fill the exact one, as such I was using the 5 x's.

Reply:
This will be resolved with [^1]. Thus I would suggest to set `Default Hostname` to `` or `upcloudobjects.com `.

[^1]: iterate-ch/cyberduck#16648